content/doc/book/blueocean/index.adoc

```diff
@@ -13,33 +13,56 @@ ifdef::backend-html5[]
+  <<blueocean/activity#activity,lists of your current and previously completed
+  Pipeline/item runs>>, as well as your Pipeline project's
+  <<blueocean/activity#branches,branches>> and any opened
+  <<blueocean/activity#pull-requests,Pull Requests>>,
+* use the <<blueocean/pipeline-run-details#,Pipeline run details view>> - where
+  you can access details (i.e. the console output) for a particular
+  Pipeline/item run, and
+* use the <<blueocean/pipeline-editor#,Pipeline Editor>> to modify Pipelines as
+  code, which are committed to source control.
+
+This chapter is intended for Jenkins users of all skill levels, but beginners
+may need to refer to some sections of the <<pipeline#,Pipeline>> chapter to
+understand some topics covered in this Blue Ocean chapter.
 
 For an overview of content in the Jenkins User Handbook, see
 <<getting-started#,User Handbook Overview>>.
 
+
 [[overview]]
 == What is Blue Ocean?
 
-Blue Ocean rethinks the user experience of Jenkins.  Designed from the ground
-up for <<pipeline#, Jenkins Pipeline>>, but still compatible with Freestyle
-jobs, Blue Ocean reduces clutter and increases clarity for every member of the
-team.
+Blue Ocean rethinks the user experience of Jenkins. Designed from the ground up
+for link:../pipeline/[Jenkins Pipeline], but still compatible with freestyle jobs,
+Blue Ocean reduces clutter and increases clarity for every member of the team.
+Blue Ocean's main features include:
 
 * *Sophisticated visualizations* of continuous delivery (CD) Pipelines,
-  allowing for fast and intuitive comprehension of pipeline's status.
-* *Pipeline editor* makes creation of Pipelines approachable by guiding the user through an intuitive and visual process to create a Pipeline.
+  allowing for fast and intuitive comprehension of your Pipeline's status.
+* *Pipeline editor* - makes creation of Pipelines approachable by guiding the
+  user through an intuitive and visual process to create a Pipeline.
 * *Personalization* to suit the role-based needs of each member of the team.
-* *Pinpoint precision* when intervention is needed and/or issues arise. Blue Ocean shows where in the pipeline attention is needed, facilitating exception handling and increasing productivity.
-* *Native integration for branch and pull requests* enables maximum developer productivity when collaborating on code with others in GitHub and Bitbucket.
+* *Pinpoint precision* when intervention is needed and/or issues arise. Blue
+  Ocean shows where in the pipeline attention is needed, facilitating exception
+  handling and increasing productivity.
+* *Native integration for branch and pull requests*, enables maximum developer
+  productivity when collaborating on code with others in GitHub and Bitbucket.
 
-To start using Blue Ocean, see **<<blueocean/getting-started#,Getting Started
-with Blue Ocean>>**.
+To start using Blue Ocean, see
+link:getting-started/[Getting started with Blue Ocean].
```

Comment:
So, is your intent to replace all xrefs (`<< ... >>` links) with `link:...` links? 
Do you prefer them?  
I'd rather maintain consistency as much as possible.
If we're going to switch, could we switch them all in one PR and add something to the style guide?  I'm honestly asking.  If you'd rather make it incrementally I'm open to that, I just want to understand that plan.

Reply:
Actually, I've been starting to switch the xref links between '.adoc' pages from the inter-document link syntax `<<...#,...>>` to the `link:...[...]` syntax based on that chat/discussion we had with @rtyler . For intra-document links (i.e. I gather this means links within a single '.adoc' page to other anchor points on the same page), I'm using the `<<...,...>>` syntax.

Please let me know if I've grabbed the wrong end of the stick here? If so, I'll update point 2 of [WEBSITE-432](https://issues.jenkins-ci.org/browse/WEBSITE-432).

Regardless of the outcome, my intent is to ensure all inter-/intra-document links are relative, so that we can publish Git branch work to personal GH pages.

Reply:
Got the update to this - new plan:

* For intra-document links, use `<<...>>` if you want to use the target title as the linking text.
* For everything else, use `link:...[...]` syntax.

Point 2 of [WEBSITE-432](https://issues.jenkins-ci.org/browse/WEBSITE-432) has been updated (hopefully last time now ;-) ) about this change.